packages/vats/test/bootstrapTests/supports.js

```diff
@@ -26,6 +26,29 @@ export const bootstrapMethods = {
   awaitVatObject: 'awaitVatObject',
 };
 
+/**
+ * @template {PropertyKey} K
+ * @template V
+ * @param {K[]} keys
+ * @param {(key: K, i: number) => V} valueMaker
+ */
+const keysToObject = (keys, valueMaker) => {
+  return Object.fromEntries(keys.map((key, i) => [key, valueMaker(key, i)]));
+};
+
+/**
+ * AVA's default t.deepEqual() is nearly unreadable for sorted arrays of strings.
```

Comment:
👏 I was just thinking yesterday of whether to propose that marcher for Ava. I bet they'd say to keep it in a separate package. Fortunately they're considering a custom marchers mechanism to add ones to `t`.